README.md

```diff
@@ -235,6 +240,17 @@ This formats `.json` files from folder with extracted archive.
 $ find . -type f -name '*.json' -print | while read line; do cat "$line" | jq > "$line.tmp" && mv "$line.tmp" "$line"; done
 ```
 
+### Obfuscating an archive
+
+You can run obfuscation with an archive by running the next command:
+
+```sh
+go run ./cmd/obfuscate-archive/main.go YOUR_ARCHIVE.tar.gz
```

Comment:
missing `$`

Reply:
Why? It's so inconvenient to copy-paste when there's `$` sing in the beginning..

Reply:
Just to keep as the other commands or remove it from all.

Reply:
Fixed, also added some syntax highlighting

Reply:
Nice. Thank you.